include/geode/inspector/topology/brep_blocks_topology.h

```diff
@@ -23,15 +23,42 @@
 
 #pragma once
 
+#include <string>
+#include <utility>
+#include <vector>
+
+#include <absl/types/optional.h>
+
 #include <geode/inspector/common.h>
+#include <geode/inspector/information.h>
 
 namespace geode
 {
+    struct ComponentMeshVertex;
     class BRep;
+    struct uuid;
 } // namespace geode
 
 namespace geode
 {
+    struct opengeode_inspector_inspector_api BRepBlocksTopologyInspectionResult
+    {
+        ProblemInspectionResult< uuid > blocks_not_meshed{
+            "uuids of block without mesh."
```

Comment:
block or Block? Should we be close to the C++ syntax or not?

Reply:
I think block is fine... but as yo want.